index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <title>Sebastián's Portfolio</title>
+</head>
+<body>
+    <div class="wrapper">
+        <nav class="flex-nav">
+            <ul>
+                <li></li>
+                <li><a href="#" class="toggleNav">☰</a></li>
+            </ul>
+        </nav>
+        <header class="name">
+            <h1>Sebastián<br>Muñoz</h1>
```

Comment:
I suggest the main header be moved to the section below as stated in the project requirements